Sources/Diagnose/SourcekitdRequestCommand.swift

```diff
@@ -56,10 +56,9 @@ public struct SourceKitdRequestCommand: AsyncParsableCommand {
       let requestInfo = try RequestInfo(request: requestString)
 
       let lineTable = LineTable(requestInfo.fileContents)
-      if let offset = lineTable.utf8OffsetOf(line: line - 1, utf8Column: column - 1) {
```

Comment:
I'm not sure it's worth it, but maybe we should keep a `nil` returning variant? In this case in particular, it would make sense to display an error about invalid line/col (though... we're currently just ignoring it anyway).